kernel/univ.mli

```diff
@@ -319,15 +319,24 @@ module AUContext :
 sig 
   type t
 
+  val repr : t -> UContext.t
+  (** [repr ctx] is [(Var(0), ... Var(n-1) |= cstr] where [n] is the length of
+      the context and [cstr] the abstracted constraints. *)
+
   val empty : t
+  val is_empty : t -> bool
 
+  (** Don't use. *)
   val instance : t -> Instance.t
-                        
+
```

Comment:
Actually, it's still used a lot, and for relatively good reason to avoid generating fresh levels (e.g. to compute canonical projections). Shouldn't it rather be a use with care or you think we could get rid of the remaining uses?